_config.yml

```diff
@@ -13,4 +13,4 @@ permalink: pretty
 paginate: 5
 exclude: ["less","node_modules","Gruntfile.js","package.json","README.md"]
 
-plugins: [jekyll-paginate, jekyll-feed]
+plugins_dir: [jekyll-paginate, jekyll-feed]
```

Comment:
What version of jekyll are you using? ``` bundle info jekyll ```
Because with version 3.6.2 (latest) I don't get any deprecated warnings.

Reply:
you are right, I have an older jekyll 3.4.3, updating